src/main/java/org/fundacionjala/coding/Dna.java

```diff
@@ -0,0 +1,18 @@
+package org.fundacionjala.coding;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class Dna {
+
+    /**
+     *
+     * @param dna ingresa el dna .
+     * @return retorna el dna que se remplazo pero sin afectar a la cadena principal.
+     */
+    public String makeComplemet(String dna) {
+        String dnaux = dna.replace("A", "t").replace("T", "a").replace("C", "g").replace("G", "c").toUpperCase();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dnaux". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)